__tests__/endpoint.test.js

```diff
@@ -390,4 +391,14 @@ describe('/api/comments/:comment_id', () => {
          expect(response.body.msg).toBe('comment not found');
        });
    });
+})
+describe('/api/users', () => {
+  test("GET:200 sends an array of topics to the client", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users).toEqual(usersData)
```

Comment:
I think it would be better to check the length of the response and that the userdata has the keys you want, that would give the tests more meaning and make them more useful to others. If you accidentally changed the test data, this would always pass